### PR TITLE
Implement My Feed Search, Filtering, and Pagination - Backend

### LINKED ISSUE #107: Implement My Feed Search, Filtering, and Pagination
Our "My Feed" page needs to be bold and robust with superb searching and filtering capabilities and good pagination.

closes #108 
closes #302 

#### Perceived Problem

- Users can't search or filter!
- Pages can't paginate!

#### Ideas / Proposed Solution(s)

- Build out the above functionality!

### PR DESCRIPTION
**Pull request by robin-macpherson**
## Description

**Issue:** #107 

Our "My Feed" page needs to be bold and robust with superb searching and filtering capabilities and good pagination.
This PR implements this functionality!

Also fixes our backend `.tsconfig.json` file finally :)

## Subtasks

- [x] I have added this PR to the Journaly Kanban project ✅
- [x] Implement filters
- [x] Implement string search
- [x] Implement pagination
- [x] Implement language query

### CODE REVIEW

File: backend/src/graphql.ts [hunk 1]
```diff
@@ -117,12 +115,56 @@ schema.queryType({
...
+            filterClauses.push({
+              OR: [
+                {
+                  title: {
+                    contains: args.search,
+                  },
+                },
+                {
+                  body: {
+                    contains: args.search,
```
Comment (Lanny) [inline backend/src/graphql.ts:155]: As we get more posts, `LIKE` queries can be very taxing on the DB without an index, and title and body aren't indexed (I think) so we should add an issue or something to the kanban as a reminder to add indexes to these (probably after we have a path forward on the case sensitivity thing)

File: backend/src/graphql.ts [hunk 2]
```diff
@@ -117,12 +115,56 @@ schema.queryType({
...
+                  },
+                },
+              ],
+            })
+          }
+
           return ctx.db.post.findMany({
             where: {
-              status: args.status as any,
```
Comment (Lanny) [inline backend/src/graphql.ts:124]: just noticed status fell out of the query here. We probably shouldn't take it as an arg and just enforce search to published posts (I assume we only want authors to be able to read draft posts, plenty of other places we need to add this check)

File: backend/src/graphql.ts (hunk 1 above)
Comment (robin-macpherson) [reply backend/src/graphql.ts:155]: Issue created! See #111

File: backend/src/graphql.ts (hunk 2 above)
Comment (robin-macpherson) [reply backend/src/graphql.ts:124]: Ok removed from args and we can chat about other use cases later

**Pull request merged**